README.md

```diff
@@ -233,6 +245,8 @@ The [Hyrax Management Guide](https://github.com/samvera-labs/hyrax/wiki/Hyrax-Ma
 * Audiovisual transcoding with `ffmpeg`
 * Setting up administrative users
 * Metadata customization
+* Virus checking
```

Comment:
Moved/reworked these from CC README to Hyrax Mgmt Guide so as to keep the README smaller.